### ISSUE TITLE
Accessibility feedback - Usability Testing Prep - eBenefits [CH31]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#15225]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

VSA: Jennifer
VSP: Trevor

### Must

1. Since the eligibility criteria for CH31 requires a disability, recruiting **must** prioritize Veterans with a service-connected disability. Since the plan is to conduct 5 sessions, at least 3 must have a service-connected disability of 20% or more, with at least one with a rating of 80% or greater.

1. Due to the above requirement, team **must** be prepared to pivot to a learning session if the participant is unable to use the prototype. It may be useful to indicate in the participant criteria language similar to the following :

> For this session, we are unable to include:
> • Those who use assistive technology, such as screen readers (JAWS, NVDA, VoiceOver, and ChromeVox).
> • People with significant vision loss.
> • People who use magnification tools.

### Should

1. Comments provided in the prototype. I only documented in the first non-signed in pass through. There is one question about the Confirmation screen, as the hierarchy of the information is confusing. This may be something to review with Design (Emily and/or Shawna).

### Consider

1. The team availability for research prioritizes east coast — **consider** how to include some times that are inclusive of west coast Veterans, in order to have diversity of perspectives (Shawna, Josh, or I may be able to help with the later hours).

1. Why are dependents able to participate in the research? Is this for the use case where they may help a Veteran?

## Relevant information

> You are entitled to services if: You have a 20 percent service-connected disability AND • You have an employment handicap. If your service-connected disability is only 10 percent, you must have a serious employment handicap to be eligible.
